contents/blog/aarrr-pirate-funnel.md

```diff
@@ -190,57 +176,38 @@ AARRR is sometimes considered a "marketing framework". This is nonsense. It span
-| Referral     | Recommends to a friend                               | All                        |
-| Referral     | Reviews product                                      | Marketing                  |
-
-As you can see, there is overlap at all levels of the funnel. "Recommending to a friend" is a quality metric that all teams influence through their work. 
+Here's how our earlier breakdown might look like if we assigned different PostHog teams to each user status:
+
+![aarrr funnel team](../images/blog/aarrr-pirate-funnel/aarrr-funnel-team.png)
+
+As you can see, there is overlap at all levels of the funnel, and we could go further. Activation is influenced by whether people coming into the funnel match our Ideal Customer Profile (ICP), so marketing also influences it even if they don't own activation directly. 
 
 ### 4. Research, test and iterate
 
 This is where effective analytics become essential. Your pirate metrics identify problems and opportunities, but you need to drill down deeper to understand the causes. 
 
-Use session recordings to observe real users using your product, develop solutions and A/B test them to test their effectiveness.
+Use session replays to observe real users using your product, develop solutions, and A/B test them to test their effectiveness.
 
-Once you've found good solutions, roll them out to your users and measure the results. It's best to roll our your solution to a subset of users first to make sure they have the expected impact – use [feature flags](/product/feature-flags) to do this.
+Once you've found good solutions, roll them out to your users and measure the results. It's best to roll out your solution to a subset of users first to make sure they have the expected impact – use [feature flags](/product/feature-flags) to do this.
 
 Dave McClure's original framework suggests you spend 80% of your effort on refining and improving existing features, and 20% on new ones. It's a good rule of thumb.
 
 > 💡 **PostHog Tip:** You can use PostHog at every stage of this journey. Use product analytics to track user behavior, session recording to observe and identify real user pain points, experimentation to run A/B tests and multivariate tests, and feature flags to stagger feature releases.
  
-## Takeaways and further reading
+## Takeaways
 
-1. There's a reason pirate metrics is still popular 15 years since its inception.  It's simple on the surface, but unlocks great depths of insight into user behavior, products and their trajectory. It's a not a cure-all for success, but it provides clarity and alignment for all teams in a startup.
+1. There's a reason pirate metrics is still popular 15 years since its inception. It's simple on the surface, but unlocks great depths of insight into user behavior, products and their trajectory. It's a not a cure-all for success, but it provides clarity and alignment for all teams in a startup.
```

Comment:
```suggestion
1. There's a reason pirate metrics is still popular 15 years since its inception. It's simple on the surface, but unlocks deep insights into user behavior, products and their trajectory. It's a not a cure-all for success, but it provides clarity and alignment for all teams in a startup.
```